### PR TITLE
FR: Corrected/Translated files

### PR DESCRIPTION
**Pull request by UnLaitQuiDanse**
Following files have been translated to french: 
ui_mag.csv | corrected somes translate errors
bn_000006.csv | File translated
ui_title_text.csv | Lines 13, 25-29, 38-41 translated
config.csv | Lines 1-50 translated
accident.csv | File translated
ao00000000.csv | File translated 
ar_accept_or010001.csv/ar_accept_or010002/ar_accept_or010003.csv/ar_accept_or010004.csv | Files translated
apc_name.csv | Lines 1-25 translated
ar_complete.csv | File translated
co01010000.csv | File translated

### CODE REVIEW

**Comment by alama** [comment]
Please move bn_000006.csv out of the Files folder

**Comment by UnLaitQuiDanse** [comment]
Where ?

File: Files/bn_000006.csv [hunk 1]
```diff
@@ -1,2 +1,2 @@
-name01#0,"""シエラ"""
-11_voice_navi09_war_033#0,"""魔神城の消滅を確認！<br>我々の完全勝利です！"""
+name01#0,"""Sierra"""
+11_voice_navi09_war_033#0,"""Nous vous confirmons l'aneantissement de Majin Castle!<br>Nous sommes sur une victoire totale!"""
```
Comment (alama) [inline Files/bn_000006.csv]: ### Notes (EN)
* If you want to translate a file that's in the "/Files/" directory then you **MUST** [move the file into another directory](https://github.com/blog/1436-moving-and-renaming-files-on-github) before you translate it. The /Files/ directory **MUST** only contain untranslated files.
### Notes (FR)
* Si vous voulez traduire un fichier se trouvant dans le dossier "/Files/" alors vous **DEVEZ** [déplacer le fichier dans un autre dossier](https://github.com/blog/1436-moving-and-renaming-files-on-github) avant de le traduire. Le dossier /Files/ **DOIT** uniquement contenir des fichiers non traduits.

**Comment by alama** [comment]
Adding a mapping for à

**Pull request merged**